ocaml-lsp-server/test/e2e/__tests__/textDocument-signatureHelp.ts

```diff
@@ -353,57 +363,75 @@ describe_opt("textDocument/completion", () => {
               **Bold**, *Italic*, *Emphasize*, Superscript, Subscript, and \`inline code\`
               
               #### Text Alignment
-              
+          
               Centered text
+              
               Left-aligned text
+              
               Right-aligned text
               
               #### Lists
               
-              1. Ordered list item 1
-              2. Ordered list item 2
-              
-              - Unordered list item 1
-              - Unordered list item 2
-              
-              - Unordered list item 1
-              - Unordered list item 2
+              1.Ordered list item 1
+              2.Ordered list item 2
+          
+              -Unordered list item 1
+              -Unordered list item 2
+
+              -Unordered list item 1
+              -Unordered list item 2
               
               #### Code Blocks
               
               \`\`\`ocaml
-                    let square x = x * x
-                    let result = square 3
+              let square x = x * x
+              let result = square 3
               \`\`\`
-              
+
+              \`\`\`python
+              def f():
+                return 0
+              \`\`\`
+
               #### Verbatim
               
+              \`\`\`verb
                   This text will be displayed verbatim.
                   No formatting will be applied.
+              \`\`\`
               
               #### Module List
               
-              * * *
-              ***@param*** \`x\` dividend
+              *Array
```

Comment:
Module list are still broken

Reply:
Wow, this is more involved. It didn't work previously so we can fix it later.

Reply:
I think I misunderstood the feature. The list is fixed now :)